SpatialGDK/Source/SpatialGDKEditor/Private/SpatialLaunchConfigCustomization.cpp

```diff
@@ -40,54 +42,94 @@ void FSpatialLaunchConfigCustomization::CustomizeChildren(TSharedRef<class IProp
 	{
 		TSharedPtr<IPropertyHandle> ChildProperty = StructPropertyHandle->GetChildHandle(ChildIdx);
 
-		// Layout regular properties as usual.
-		if (ChildProperty->GetProperty()->GetName() != "ServerWorkersMap")
+		if (ChildProperty->GetProperty()->GetName() == "ServerWorkersMap")
 		{
-			StructBuilder.AddProperty(ChildProperty.ToSharedRef());
-			continue;
-		}
+			// Layout ServerWorkers map in a way that does not allow resizing and key edition.
+			uint32 NumEntries;
+			ChildProperty->GetNumChildren(NumEntries);
 
-		// Layout ServerWorkers map in a way that does not allow resizing and key edition.
-		uint32 NumEntries;
-		ChildProperty->GetNumChildren(NumEntries);
-
-		IDetailGroup& NewGroup = StructBuilder.AddGroup("ServerWorkersMap", ChildProperty->GetPropertyDisplayName());
-		NewGroup.HeaderRow()
-		.NameContent()
-		[
-			SNew(STextBlock).Text(FText::FromString(TEXT("Server Workers")))
-		]
-		.ValueContent()
-		[
-			SNew(STextBlock).Text(FText::FromString(FString::Printf(TEXT("%i Elements"), NumEntries)))
-		];
-
-		for (uint32 EntryIdx = 0; EntryIdx < NumEntries; ++EntryIdx)
-		{
-			TSharedPtr<IPropertyHandle> EntryProp = ChildProperty->GetChildHandle(EntryIdx);
-			check(EntryProp != nullptr);
-			TSharedPtr<IPropertyHandle> EntryKeyProp = EntryProp->GetKeyHandle();
-			check(EntryKeyProp != nullptr);
-			
-			FName* KeyPtr = reinterpret_cast<FName*>(EntryKeyProp->GetValueBaseAddress(reinterpret_cast<uint8*>(EditedObject[0])));
-
-			IDetailGroup& Entry = NewGroup.AddGroup(*KeyPtr, FText::FromName(*KeyPtr));
-			uint32 NumEntryFields;
-			EntryProp->GetNumChildren(NumEntryFields);
-
-			for (uint32 EntryField = 0; EntryField < NumEntryFields; ++EntryField)
+			IDetailGroup& NewGroup = StructBuilder.AddGroup("ServerWorkersMap", ChildProperty->GetPropertyDisplayName());
```

Comment:
If I'm not mistaken, Jen removed the ServerWorkersMap from the launch configuration, so all this bit can be removed.

Reply:
Ah true, will remove.